nservicebus/msmq/dead-letter-queues.md

```diff
@@ -15,3 +15,23 @@ NOTE: If DLQ is enabled messages will remain in the senders outgoing queue until
 
 
 partial: config
+
+
+### Monitoring DLQ
+
+Misconfiguration of [routing](/nservicebus/messaging/routing) may result in messages ending up in the DLQ. It is very important to monitor the DLQ in order to detect potential routing configuration errors or other situations that may lead to messages being moved to the dead letter queue.
+
+NOTE: While there is usually a single error queue for the entire system, each machine has a separate dead letter queue. This means that the DLQ on each machine has to be monitored individually.
```

Comment:
Perhaps spell out that its a central "NServiceBus error queue"  to avoid readers thinking that msmq has a error queue concept?